azurerm/disks.go

```diff
@@ -0,0 +1,40 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+type UnmanagedDiskMetadata struct {
```

Comment:
they should be private, however this is a WIP before refactoring (and placing this somewhere in `helpers/`)

Reply:
(also, we tend to rely directly on the SDK Structs rather than composing them ourselves, since this is information which should be provided by the SDK but isn't at this point)